test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -702,6 +703,16 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 			}
 		})
 
+		It("should have core kube-system addons running the correct version", func() {
```

Comment:
This is nice, we should probably guard this in a `if eng.ExpandedDefinition.Properties.OrchestratorProfile.KubernetesConfig.CustomKubeProxyImage == "" {` block

Reply:
good call